### ISSUE TITLE
Change base colors to improve contrast

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5293]
Change any colors that don't match the Dart branding guidelines. This results in multiple violations of [WCAG 2.0 G18](https://www.w3.org/TR/WCAG20-TECHS/G18.html) and GAR criteria 1.4.
